### PR TITLE
DB context tests

### LINKED ISSUE #107: Backend: Test for BmtDbContext model connections

### PR DESCRIPTION
**Pull request by Christdej**
Closes #107

### CODE REVIEW

File: backend/tests/DbContext.cs [hunk 1]
```diff
@@ -1,20 +1,75 @@
...
+        {
+            EvaluationService evaluationService = new EvaluationService(_context);
+            Evaluation exampleEvaluation = evaluationService.GetAll().First();
+            int participantsBefore = exampleEvaluation.Participants.Count;
+            ParticipantService participantService = new ParticipantService(_context);
+
+            participantService.Create("some_id_we_got_from_fusion", exampleEvaluation, Organization.Engineering, Role.Participant);
+            int participantsAfter = exampleEvaluation.Participants.Count;
+
+            Assert.True(participantsBefore + 1 == participantsAfter);
```
Comment (nicholasdalhaug) [inline backend/tests/DbContext.cs:41]: We get a better output from testing if we use `Assert.Equals` instead. Same farther down.

File: backend/tests/DbContext.cs [hunk 2]
```diff
@@ -1,20 +1,75 @@
...
             Assert.Equal(11, questions.Count());
         }
+
+
+        [Fact]
+        public void ParticipantAddedToEvaluation()
+        {
+            EvaluationService evaluationService = new EvaluationService(_context);
+            Evaluation exampleEvaluation = evaluationService.GetAll().First();
+            int participantsBefore = exampleEvaluation.Participants.Count;
```
Comment (nicholasdalhaug) [inline backend/tests/DbContext.cs:35]: I think a newline right here, to follow the AAA(Arrange, Act, Assert) pattern would be nice.

File: backend/tests/DbContext.cs [hunk 3]
```diff
@@ -1,20 +1,75 @@
...
+
+        [Fact]
+        public void ParticipantAddedToProject()
+        {
+            ProjectService projectService = new ProjectService(_context);
+            Project exampleProject = projectService.GetAll().First();
+            Evaluation exampleEvaluation = exampleProject.Evaluations.First();
+
+            int participantsBefore = exampleProject.Evaluations.First().Participants.Count;
+            ParticipantService participantService = new ParticipantService(_context);
```
Comment (nicholasdalhaug) [inline backend/tests/DbContext.cs:52]: The participant service could be initialized before the int, thereby also using the AAA(Arrange, Act, Assert) pattern.

**Review by nicholasdalhaug — CHANGES_REQUESTED** [review]

File: backend/tests/DbContext.cs (hunk 1 above)
Comment (Christdej) [reply backend/tests/DbContext.cs:41]: Good point, will fix =)

File: backend/tests/DbContext.cs (hunk 2 above)
Comment (Christdej) [reply backend/tests/DbContext.cs:35]: Done

File: backend/tests/DbContext.cs (hunk 3 above)
Comment (Christdej) [reply backend/tests/DbContext.cs:52]: Done

**Review by ReedOnly — APPROVED** [review]

**Review by nicholasdalhaug — APPROVED** [review]

**Pull request merged**